pkgs/os-specific/bsd/freebsd/pkgs/libcasper.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  stdenv,
+  mkDerivation,
+  libnv,
+}:
+mkDerivation {
+  path = "lib/libcasper/libcasper";
```

Comment:
We removed libcasper before, and now we're adding it back?

Reply:
You're referring to the earlier diff in this PR? that's for the compat generation, which is separate from the actual package (this) which is a dependency of various things. Compat is just for building packages for the build machine.

Reply:
I updated the commit message for the compat patch indicating this.